.gitignore

```diff
@@ -17,5 +17,10 @@ tmp
 .DS_Store
 .idea
 
-## Pug ##
-*.html
+## Pug and Babel ##
+app/*.html
+app/*/*.html
+app/*/*/*.html
+app/*.js
```

Comment:
maybe you should change this gitignore paths.... to only ingore angular istead

Reply:
This is ready in books-bootstrap